frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -359,7 +361,20 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
                     await breakpoint(400) // Debounce for lots of quick filter changes
 
                     const startTime = performance.now()
-                    const response = await api.recordings.list(params)
+                    let response
+
+                    try {
+                        response = await api.recordings.list(params)
+                    } catch (error: any) {
+                        captureException(error)
+                        actions.loadSessionRecordingsFailure(error.message, error)
+
+                        return {
+                            has_next: false,
+                            results: [],
+                        }
+                    }
```

Comment:
If the exception isn't handled in this code section, it will be processed by the `loadersPlugin` in `initKea.ts`. This will trigger the `onFailure` function, which already includes error handling with `lemonToast.error`, resulting in a duplicate error toast.

For reference, here’s the relevant code:
https://github.com/PostHog/posthog/blob/145cacf60aaedcc077911f7d3231bd6efb70072c/frontend/src/initKea.ts#L102-L104

I’ve added exception handling above to prevent the duplication of error toasts.